x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -542,6 +543,15 @@ export const uiSettings: Record<string, UiSettings> = {
     schema: schema.number({ min: 0, max: 100 }),
     requiresPageReload: true,
   },
+  [apmEnableTransactionProfiling]: {
```

Comment:
Should this setting also appear in APM's settings?

Reply:
It's already there: https://github.com/elastic/kibana/pull/176302/files#diff-5de8b7d31f5c6fcb6dc43cc4394e2d42788cdabf00faf69a41e5c443462420e2R62